jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultipleInputsTests.java

```diff
@@ -0,0 +1,260 @@
+        String surname = inputData.get(2);
+        multipleInputs.sendKeys(locator, surname);
+        assertEquals(multipleInputs.getText(locator), surname);
+    }
+
+    @Test
+    public void sendAllKeysTest() {
+        multipleInputs.clearAll();
+
+        multipleInputs.sendKeysAll(inputDataList);
+        assertEquals(multipleInputs.getAllTexts(), inputDataList);
+    }
+
+    @Test
+    public void clearTest() {
+        String value = inputData.get(1);
+        multipleInputs.sendKeys(value);
+        multipleInputs.clear();
+        assertEquals(multipleInputs.getText(), "");
+
+        int index = 2;
+        String name = inputData.get(index);
+        multipleInputs.sendKeys(index, name);
+        multipleInputs.clear(index);
+        assertEquals(multipleInputs.getText(index), "");
+
+        String locator = "#mi-i-2";
+        String surname = inputData.get(2);
+        multipleInputs.sendKeys(locator, surname);
+        multipleInputs.clear(locator);
+        assertEquals(multipleInputs.getText(locator), "");
+    }
+
+    @Test
+    public void clearAllTest() {
+        multipleInputs.sendKeysAll(inputDataList);
+        multipleInputs.clearAll();
+
+        multipleInputs.getAllTexts().forEach(t -> assertEquals(t, ""));
+    }
+
+    @Test
+    public void focusTest() {
+        multipleInputs.focus();
+        assertTrue(isElementInViewPort(multipleInputs.core()));
+        redButton.hover();
+
+        int index = 2;
+        multipleInputs.focus(index);
+        assertTrue(isElementInViewPort(multipleInputs.core()));
+        redButton.hover();
+
+        String locator = "#mi-i-1";
+        multipleInputs.focus(locator);
+        assertTrue(isElementInViewPort(multipleInputs.core()));
+    }
+
+    @Test
+    public void placeholderTest() {
+        assertEquals(multipleInputs.placeholder(), "");
+        assertEquals(multipleInputs.placeholder(1), "");
+        assertEquals(multipleInputs.placeholder("#mi-i-2"), "");
+    }
+
+    @Test
+    public void placeholderAllTest() {
+        multipleInputs.placeholderAll().forEach(p -> assertEquals(p, ""));
+    }
+
+    @Test
+    public void inputTest() {
+        multipleInputs.clearAll();
+
+        String value = inputData.get(1);
+        multipleInputs.input(value);
+        assertEquals(multipleInputs.getText(), value);
+
+        multipleInputs.clearAll();
+        int index = 2;
+        String name = inputData.get(index);
+        multipleInputs.input(name, index);
+        assertEquals(multipleInputs.getText(index), name);
+
+        multipleInputs.clearAll();
+        String locator = "#mi-i-2";
+        String surname = inputData.get(2);
+        multipleInputs.input(surname, locator);
+        assertEquals(multipleInputs.getText(locator), surname);
+    }
+
+    @Test
+    public void inputAllTest() {
+        multipleInputs.clearAll();
+
+        multipleInputs.inputAll(inputDataList);
+        assertEquals(multipleInputs.getAllTexts(), inputDataList);
```

Comment:
it's better to write something like to use jdi-like code instead of testng: 
multipleInputs.is().texts(inputDataList);